cli/napi/threadsafe_functions.rs

```diff
@@ -35,6 +35,22 @@ impl TsFn {
     Ok(())
   }
 
+  pub fn ref_(&mut self) -> Result {
+    self
+      .tsfn_sender
+      .unbounded_send(ThreadSafeFunctionStatus::Alive)
+      .map_err(|_| Error::GenericFailure)?;
+    Ok(())
+  }
+
+  pub fn unref(&mut self) -> Result {
+    self
+      .tsfn_sender
+      .unbounded_send(ThreadSafeFunctionStatus::Dead)
+      .map_err(|_| Error::GenericFailure)?;
```

Comment:
This is brittle and not idempotent - calling `unref()` multiple times will make `NapiState.active_threadsafe_functions` go negative. I was looking into fixing this and I believe the most bulletproof solution would either be implementing something similar to what @aapoalas did for FFI functions, or make `TsFn` have ID and store a hashmap of TsFn mappings in `NapiState` that keeps track of reference counts. WDYT?